ecommerce/serializers.py

```diff
@@ -33,8 +35,18 @@ def get_course_runs(self, instance):
             CourseRunSerializer(instance=course_run).data for course_run in course_runs
         ]
 
+    def get_thumbnail(self, instance):
+        """Return the thumbnail for the course or program"""
+        content_object = instance.product.content_object
+        try:
+            if isinstance(content_object, CourseRun):
+                return content_object.course.thumbnail.url
+            return content_object.thumbnail.url
+        except ValueError:
```

Comment:
Is there a better way to handle this than `ValueError` that we might actually want to have raised? More code might be better if the failure state is to just show the default image.

Reply:
I updated this to check if `thumbnail` is falsey instead